openeo/rest/job.py

```diff
@@ -424,6 +418,43 @@ def load_bytes(self) -> bytes:
     # TODO: more `load` methods e.g.: load GTiff asset directly as numpy array
 
 
+    def _download_to_file(self, url: str, target: Path, chunk_size: int):
+        head = self.job.connection.head(url, stream=True)
+        if head.ok and 'Accept-Ranges' in head.headers and 'bytes' in head.headers['Accept-Ranges']:
+            file_size = int(head.headers['Content-Length'])
```

Comment:
I guess you should also check that there is a "Content-Length" header, otherwise this will fail